06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md

```diff
@@ -0,0 +1,7 @@
+#### Output
+- **Using**: GitHub
+- **Deliver**: the repository for my proposed project and at least one `Pull Request`
+- **With**: *independently*
+
+#### Github repo
+- [andela-hchukwu/Inverted-Index](https://github.com/andela-hchukwu/Inverted-Index.git)
```

Comment:
Nice setup!

I however can't merge this as the other output isn't done properly yet.

Kindly address!